generator/mavschema.xsd

```diff
@@ -16,7 +16,17 @@
         </xs:restriction>
     </xs:simpleType>
 </xs:attribute>
+<xs:attribute name="increment" type="xs:float"/> <!-- param -->
 <xs:attribute name="index" type="xs:unsignedByte"/> <!-- param -->
+<xs:attribute name="isDestination" type="xs:boolean" default="true"/> <!-- param -->
+<xs:attribute name="hasLocation" type="xs:boolean" default="true"/> <!-- param -->
+<xs:attribute name="decimalPlaces" type="xs:unsignedByte"/> <!-- param -->
```

Comment:
decimal places should probably be signed. this would allow for i.e. both milimeters (-3) and kilometers (3)